src/Workspaces/Core/Portable/Workspace/Solution/SolutionState.CompilationTracker.cs

```diff
@@ -811,6 +827,26 @@ public CompilationInfo(Compilation compilation, bool hasSuccessfullyLoaded, Immu
                 {
                     throw ExceptionUtilities.Unreachable;
                 }
+
+                // Local functions
+                static SourceGeneratedDocumentState? FindExistingGeneratedDocumentState(
```

Comment:
```suggestion
                return;

                // Local functions
                static SourceGeneratedDocumentState? FindExistingGeneratedDocumentState(
```

Reply:
The `return` in this case would produce an unreachable statement warning.

Reply:
fun :)